aiida_orca/parsers/__init__.py

```diff
@@ -24,25 +28,25 @@ def parse(self, **kwargs):
         If it would be an optimization run, the relaxed structure also will
         be stored under relaxed_structure key.
         """
-
         try:
             out_folder = self.retrieved
         except NotExistent:
             return self.exit_codes.ERROR_NO_RETRIEVED_FOLDER
 
-        fname_out = self.node.process_class._OUTPUT_FILE  #pylint: disable=protected-access
-        fname_relaxed = self.node.process_class._RELAX_COORDS_FILE  #pylint: disable=protected-access
-        # fname_traj = self.node.process_class._TRAJECTORY_FILE  #pylint: disable=protected-access
-        # fname_hessian = self.node.process_class._HESSIAN_FILE  #pylint: disable=protected-access
+        process_cls = self.node.process_class
+        fname_out = process_cls._OUTPUT_FILE  # pylint: disable=protected-access
+        fname_relaxed = process_cls._RELAX_COORDS_FILE  # pylint: disable=protected-access
 
-        if fname_out not in out_folder._repository.list_object_names():  #pylint: disable=protected-access
+        if fname_out not in out_folder.list_object_names():
             raise OutputParsingError('Orca output file not retrieved')
 
         try:
-            with self.retrieved.open(fname_out) as handler:
-                parsed_obj = ccread(handler.name)
+            with self.retrieved.open(fname_out, 'rb') as handle:
+                with tempfile.NamedTemporaryFile('w+b') as tmpfile:
+                    shutil.copyfileobj(handle, tmpfile)
+                    parsed_obj = ccread(tmpfile.name)
                 parsed_dict = parsed_obj.getattributes()
-        except OutputParsingError:  #pylint: disable=bare-except
+        except OutputParsingError:
```

Comment:
Could you please change to
```suggestion
        except Exception:
```

The original code was wrong. `OutputParsingError` is an AiiDA type, and is never raised in the `ccread` function.
Parsing here can fail in variety of ways (ccread was extracted from the cclib library) and we need to catch them all. Would be better to be more specific in the future...

Reply:
Done